tests/run.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-nvim --headless -u tests/init.lua -c "PlenaryBustedDirectory tests/ {minimal_init = 'tests/init.lua', sequential = true}"
+# The Tests Runner runs tests for all the environments
```

Comment:
@peterfication this is a draft, the idea is to clone the repos of plugins and use what is required isolated according to the tested features, like Telescope

Reply:
We probably need some kind of vim setup inside a spec file then, right?